app/views/macros/search-results.njk

```diff
@@ -0,0 +1,30 @@
+{% macro searchResults(model) %}
+  <div>
+    <h2 class="govuk-heading-s" id='totalResults' {{ model.total }} results>{{ model.total }} results</h2>
+    <hr class="govubk-section-break govuk-section-reak--visible section-break-heading">
+    {% for item in model.results %}
+      <div>
+        <div class='grid-column-two-one'>
+          <h3 class='margin-top-10'>
+            <a href="/" class="govuk-link govuk-link--no-visited-state">
+              {{ item.payee_name }}
+            </a>
+          </h3>
+          <div>
+            <div class="govuk-heading-s search-results-amount">{{ item.amount }}</div>
+          </div>
+        </div>
+        <div class="govuk-body subtitle margin-bottom-20 govuk-!-font-size-16">Location: {{ item.part_postcode }}, {{item.town}}, {{item.county_council}}</div>
+        <hr class="govubk-section-break govuk-section-reak--visible section-break-heading--lightgray">
+      </div>
+    {% else %}
```

Comment:
are we combining if else block from different file? is it good for readability?

Reply:
This is a nunjucks thing called for else block. Very bizarre, I know but its supported by the engine, https://mozilla.github.io/nunjucks/templating.html#for